src/interpreter/plugin/MedianPlugin.ts

```diff
@@ -51,4 +66,38 @@ export class MedianPlugin extends FunctionPlugin {
       }
     })
   }
+
+  public large(ast: ProcedureAst, formulaAddress: SimpleCellAddress): InternalScalarValue {
```

Comment:
median is actually different (even conceptually), due to some ugly hacks on emptyValues

not sure if it is worth it to abstract from two functions (small/large), since we are not going probably to have more of those

Reply:
Right, I missed the difference with emptyValues. So indeed, maybe there is no point in changing anything here